### DISCUSSION TITLE
What's the voice that disables the Search field in the iot xml?

### DISCUSSION THREAD

**Discussion by charger2000** [memstechtips/UnattendedWinstall#126] [Q&A]
HI, as stated, what's the voice that disables the Search field in the application bar in the iot xml? I'd like to leave it there, thanks

**Comment by memstechtips** [comment]
Can you post a screenshot of the issue?

**Comment by charger2000** [comment]
I don't have access to that PC now, what I mean is that after using the iot xml unattended file the search field in the Windows 10 task bar is disabled, I have to right click the start button and click on "Search". I'd like to re-enable the default search field on the task bar, and, preferably, remove that from the xml file

thanks

**Comment by memstechtips** [comment]
Hey there and sorry for the late reply,

Since the creation of this issue I have released a new version of the XML file and a major update to [Winhance](https://winhance.net/), I recommend that you watch [this video](https://youtu.be/lrq3ph3xi50) which provides a full demo of the current version along with how to use the new version of WIMUtil and how to install the drivers etc. You now also have the ability to create an autounattend.xml file based on your selections in Winhance, so you have more control over which apps are removed and which settings are applied.

I have also created [this video](https://youtu.be/sCd1TI6AbOI) about v3.0.0 of UnattendedWinstall, I suggest that you watch it too if you're interested.

In the new version, this should be fixed.

I'm closing this issue now, but if you need more help with the same issue, feel free to reopen it.
